awx/ui/test/e2e/tests/test-workflow-visualizer.js

```diff
@@ -50,115 +54,151 @@ module.exports = {
 
         Promise.all(resources)
             .then(([source, template, project, workflow]) => {
-                data = { source, template, project, workflow };
+                testActionsWorkflowtemplateLink = `a[href="/#/templates/workflow_job_template/${workflow.id}"]`;
+                testActionsProject = `#projects_table tr[id="${project.id}"]`;
+                testActionsJobTemplate = `#templates_table tr[id="${template.id}"]`;
+
+                console.log(testActionsWorkflowtemplateLink);
+
                 done();
             });
+    },
+    'navigate to the workflow template visualizer': client => {
         client
             .login()
             .waitForAngular()
             .resizeWindow(1200, 1000)
-            .useXpath()
             .findThenClick(workflowTemplateNavTab)
             .pause(1500)
             .waitForElementNotVisible(spinny)
-            .clearValue(workflowSearchBar)
-            .setValue(workflowSearchBar, [workflowText, client.Keys.ENTER])
-            .waitForElementVisible(workflowSearchBadgeCount)
-            .waitForElementNotVisible(spinny)
-            .findThenClick(workflowSelector)
-            .findThenClick(workflowVisualizerBtn);
+            .search(workflowSearchBar, workflowText)
+            .waitForElementPresent(templatesListBadge)
+            .waitForElementVisible(templatesListBadge);
+
+        client.expect.element(templatesListBadge).text.to.equal('1').before(10000);
+
+        client
+            .findThenClick(testActionsWorkflowtemplateLink)
+            .findThenClick(workflowTemplateVisualizerBtn);
     },
     'verify that workflow visualizer root node can only be set to always': client => {
         client
-            .useXpath()
             .findThenClick(rootNode)
-            .clearValue(projectSearchBar)
-            .setValue(projectSearchBar, [testActionsProjectText, client.Keys.ENTER])
-            .pause(1000)
-            .findThenClick(testActionsProject)
-            .findThenClick(edgeTypeDropdownBar)
-            .waitForElementNotPresent(successDropdown)
-            .waitForElementNotPresent(failureDropdown)
-            .waitForElementPresent(alwaysDropdown);
-    },
-    'verify that a non-root node can be set to always/success/failure': client => {
+            .search(jobTemplateSearchBar, testActionsJobTemplateText)
+            .findThenClick(testActionsJobTemplate);
+
         client
             .useXpath()
-            .findThenClick(childNode)
-            .pause(1000)
-            .waitForElementNotVisible(spinny)
             .findThenClick(edgeTypeDropdownBar)
-            .waitForElementPresent(successDropdown)
-            .waitForElementPresent(failureDropdown)
+            .waitForElementNotPresent(successDropdown)
+            .waitForElementNotPresent(failureDropdown)
```

Comment:
wait, why did this one change so the success and failure options were not there?

Reply:
@dsesami We are rolling out the test updates for workflow convergence directly onto the feature branch so I'm closing this PR out.